packages/hoppscotch-common/src/components/collections/graphql/Collection.vue

```diff
@@ -358,7 +359,15 @@ const toggleShowChildren = () => {
   showChildren.value = !showChildren.value
 }
 
-const removeCollection = () => {
+const handleTokenValidation = async () => {
+  const { valid, error } = await isValidUser()
+  if (!valid) toast.error(error)
+  return valid
+}
```

Comment:
since we are using this function in multiple files it's better to move this to a file and import it.

```
export const handleTokenValidation = async () => {
  const toast = useToast()

  const { valid, error } = await isValidUser()
  if (!valid) toast.error(error)
  return valid
}

```